pom.xml

```diff
@@ -1573,7 +1572,7 @@
         <swagger-core.version>2.1.2</swagger-core.version>
         <swagger-parser-groupid.version>io.swagger.parser.v3</swagger-parser-groupid.version>
         <swagger-parser.version>2.0.26</swagger-parser.version>
-        <testng.version>7.4.0</testng.version>
+        <testng.version>7.3.0</testng.version>
```

Comment:
@agilob thanks for the PR. What's the issue in using v7.4.0?

Reply:
with testng 7.4 and surefire dont play well with `parallel` flag https://stackoverflow.com/questions/68143219/maven-surefire-failsafe-testng-7-4-0-parallel-error-void-org-testng-xml-xmlsuit

Reply:
👌